app/routers/user.py

```diff
@@ -92,3 +96,55 @@ def get_all_user_events(session: Session, user_id: int) -> List[Event]:
         session.query(Event).join(UserEvent)
         .filter(UserEvent.user_id == user_id).all()
     )
+
+
+@router.post("/enable")
+def enable_current_user(
+        request: Request, session: Session) -> bool:
+    """function to enable the user.
+    after successful enable it will be directed to main page.
+    if the disable fails user will stay at settings page
+    and an error will be shown."""
+    disable_successful = enable(session, get_current_user)
+    if disable_successful:
+        # disable succeeded- the user will be directed to homepage.
+        app.url_path_for("/")
+
+    else:
+        # if disable wasn't successful, user will stay in the
+        # settings page and an error message will show.
+        pass
+
+
+@router.post("/disable")
+def disable_logged_user(
+        request: Request, session: Session) -> bool:
+    """route that sends request to disable the user.
+    after successful disable it will be directed to main page.
+    if the disable fails user will stay at settings page
+    and an error will be shown."""
+    disable_successful = disable(session, get_current_user)
+    if disable_successful:
+        # disable succeeded- the user will be directed to homepage.
+        app.url_path_for("/")
+    else:
+        # if disable wasn't successful, user will stay in the
+        # settings page and an error message will show.
+        pass
+
+
+@router.post("/enable")
```

Comment:
How is that different from the `/enable` above?

Reply:
above is a disabling request, this router is for enabling. used accidently the wrong function- fixed this in the current push

Reply:
oops. :/ accidently copied it instead of cutting for it to look nicer..